src/gc/gcinterface.h

```diff
@@ -463,6 +465,8 @@ class IGCHandleManager {
     virtual Object* GetDependentHandleSecondary(OBJECTHANDLE handle) = 0;
 
     virtual Object* InterlockedCompareExchangeObjectInHandle(OBJECTHANDLE handle, Object* object, Object* comparandObject) = 0;
+
+    virtual bool HandleAsyncPinnedHandles() = 0;
```

Comment:
Sounds good, will do!

Am I right in thinking that this is dead code in CoreCLR, so I won't be able to test it until it ends up in Desktop?